h/templates/emails/reply_notification.html.jinja2

```diff
@@ -1,5 +1,5 @@
 <p>
-  <a href="{{ reply_user_url }}">{{ reply_user.username }}</a>
+  <a href="{{ reply_user_url }}">{{ reply_user.rendered_name }}</a>
```

Comment:
All the templates still seem to be using `rendered_name`, despite it having changed elsewhere. Given all the tests are still passing, does this mean that this bit of functionality isn’t covered by the tests?

Reply:
Gah, sorry. That is really sloppy of me. Indeed we don't have automated tests for the output of these templates. Like many views, there are just tests that the expected context is generated for the template.

Reply:
Ah wait. We _do_ have tests that render these templates (`test_jinja_templates_render`), but it looks like they don't catch this class of error.